pkg/operator/testlib/vcsim_framework.go

```diff
@@ -155,7 +158,8 @@ folder = "/DC0/vm"
 [VirtualCenter "dc0"]
 datacenters = "DC0"
 `
-	err := gcfg.ReadStringInto(&cfg, data)
+	//err := gcfg.ReadStringInto(&cfg, data)
```

Comment:
can we drop this commented code or are we leaving it in for historical reasons?

Reply:
this will be removed yes.  merge conflict resolving is fun.